### PR TITLE
[Suggestions requested ] Portuguese for Portugal is still missing in options.  Attempts to fix  

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
I18N "Make Portugal Portugues appear in Options"

#### Purpose of change
Get new language to show up in options

#### Describe the solution

I've attempted to match the naming schema to match the po file.  I've also split out brasilian and portugul into their different microsoft language codes. Mirrors #24832 However all I'm accomplishing when I compile is removing every language but english from the language options.

#### Describe alternatives you've considered
Making better decisions with my life.

### CODE REVIEW

File: src/options.cpp [hunk 1]
```diff
@@ -4244,10 +4244,10 @@ void options_manager::update_global_locale()
             std::locale::global( std::locale( "nl_NL.UTF-8" ) );
         } else if( lang == "pl" ) {
             std::locale::global( std::locale( "pl_PL.UTF-8" ) );
+        } else if( lang == "pt" ) {
+            std::locale::global( std::locale( "pt.UTF-8" ) );
```
Comment (mqrause) [inline src/options.cpp:4248]: This should probably stay `"pt_PT.UTF-8"`

File: src/lang_stats.inc [hunk 2]
```diff
@@ -19,6 +19,7 @@
 {"pl"sv, 64266, 53249},
 {"pt"sv, 131, 117384},
 {"pt_BR"sv, 44591, 72924},
+{"pt_BR"sv, 445, 72924},
```
Comment (Maleclypse) [inline src/lang_stats.inc:22]:
```suggestion
```

File: src/options.cpp (hunk 1 above)
Comment (Maleclypse) [inline src/options.cpp:4248]:
```suggestion
            std::locale::global( std::locale( "pt_PT.UTF-8" ) );
```

**Comment by themaster567** [comment]
I believe that there needs to be a MOTD and credits file or there's a chance that most of the title screen will be blank. I propose that the PT_BR one for each is copied and pasted as a placeholder until they can be properly translated.

**Comment by Maleclypse** [comment]
> I believe that there needs to be a MOTD and credits file or there's a chance that most of the title screen and the entire credits screen will be blank. I propose that the PT_BR one for each is copied and pasted as a placeholder until they can be properly translated.

Don't those default to English when not translated?

**Comment by themaster567** [comment]
Possibly, but every other language shipped with the game has a MOTD file, so I haven't personally seen that.

**Comment by Maleclypse** [comment]
@katemonster33 I think I've figured out that there is something missing in regards to `AddGlyphRangesFromCLDRForPT` and that's causing the new portuguese dialect to not show up in the language selection and me using that line twice is probably the reason that when I compile this all languages except english are missing?  I'm just not sure what to do here since I can't really copy any other language additions as this is the first since this menu migrated to imgui.  When you have a second can you look at this?  Thanks!

**Comment by mqrause** [comment]
This PR works for me. If all languages are missing for you, then that means the po files weren't created. There's an extra script you need to run for this when you compile with VS.

**Pull request merged**